js/data.js

```diff
@@ -89,6 +89,29 @@ const _CLEARING_TYPES = {
   }
 }
 
+const _BOT_PLAYERS = {
+  MECHANICAL_MARQUISE: {
+      player: "The Mechanical Marquise",
+      faction: _FACTIONS.MARQUISE_DE_CAT,
```

Comment:
I would favor referencing it by id ("MARQUISE_DE_CAT"), instead of pointing to the actual faction. Then whoever is using the data can look it up by name later. This will set us up better for when we move to a database (just kidding, let's not actually do that, but that's what we're mimicking). That should also break the ordering dependency which this would have right now, and which might lead to confusing bugs!